db/db_filesnapshot.cc

```diff
@@ -195,24 +187,20 @@ Status DBImpl::GetLiveFilesStorageInfo(
         total_wal_size += wal->SizeFileBytes();
       }
       if (total_wal_size < opts.wal_size_for_flush) {
-        flush_memtable = false;
+        need_flush_memtable = false;
       }
       live_wal_files.clear();
     }
   }
+  return s;
+}
 
+void DBImpl::GetNonWALLiveFiles(const LiveFilesStorageInfoOptions& opts,
```

Comment:
Nit: I feel `GetLiveFilesExceptWals` is a better name.

Reply:
The function name suggests that it is about non-wal files, but `min_log_num` is an output argument. Can we remove it and put it out of this function? Can we evaluate it in `GetLiveWals()` below?

Reply:
In the original GetLiveFilesStorageInfo() , min_log_num is obtained within a mutex lock. I am not sure whether it is fine to get it later in the GetLiveWals().